### PR TITLE
docs: Document users in e2e-security demo

### PR DESCRIPTION
**Pull request by sbernauer**

### CODE REVIEW

**Comment by sbernauer** [comment]
Can not approve my own PR, but your stuff looks good @soenkeliebau

**Comment by soenkeliebau** [comment]
As discussed in the daily, added some more information to the readme: @lfrancke

**Review by soenkeliebau — APPROVED** [review]
For me this would suffice for now, @lfrancke?

**Comment by lfrancke** [comment]
I'm afraid I'm not happy just yet :(
From an accessibility standpoint there is a lot of data "hidden" in the images.
Can we at least get a rough written summary as for the other demos?
They have an `Overview` section that lists at least the tools that are deployed and some additional information.

I'm fine with not replicating everything that is in the image but at least a short description.

See the [anomaly](https://docs.stackable.tech/home/nightly/demos/spark-k8s-anomaly-detection-taxi-data) one for example.

"This demo will [....] You can see the [...]."

**Review by lfrancke — CHANGES_REQUESTED** [review]
I'm afraid I'm not happy just yet :(
From an accessibility standpoint there is a lot of data "hidden" in the images.
Can we at least get a rough written summary as for the other demos?
They have an `Overview` section that lists at least the tools that are deployed and some additional information.

I'm fine with not replicating everything that is in the image but at least a short description.

See the [anomaly](https://docs.stackable.tech/home/nightly/demos/spark-k8s-anomaly-detection-taxi-data) one for example.

"This demo will [....] You can see the [...]."

**Comment by sbernauer** [comment]
LGTM, many thanks @soenkeliebau!

File: docs/modules/demos/pages/end-to-end-security.adoc [hunk 1]
```diff
@@ -5,20 +5,26 @@
...
-* HDFS
-
-
-Install this demo on an existing Kubernetes cluster:
+This demo will:
+
+* Install the Stackable operators
+* Spin up the following data products
+** *Trino*: A fast distributed SQL query engine for big data analytics that helps you explore your data universe. This demo uses it to enable SQL access to the data.
+** *Spark*: A multi-language engine for executing data engineering, data science, and machine learning. This demo uses it to batch-process data from S3 by training and scoring an unsupervised anomaly detection model and writing the results into a Trino table. Spark uses an isolation forest algorithm from the scikit-learn machine learning library in this demo.
```
Comment (lfrancke) [inline docs/modules/demos/pages/end-to-end-security.adoc:15]: Is this correct? Does this do the scikit-learn stuff? Or is this a copy & paste error from the anomaly demo?

Comment (sbernauer) [reply docs/modules/demos/pages/end-to-end-security.adoc]: You are right, thanks! c11b151b16a50e712741cec3e7d5bc5b976dc9ce

**Review by lfrancke — APPROVED** [review]

**Pull request merged**